src/americana.js

```diff
@@ -370,6 +381,11 @@ var bridgeLayers = [
   lyrRoad.trunkExpresswayTollBridge.surface(),
   lyrRoad.motorwayTollBridge.surface(),
 
+  lyrRail.railBridge.dashes(),
+  lyrRail.railBridge.fill(),
+  lyrRail.railServiceBridge.dashes(),
+  lyrRail.railServiceBridge.fill(),
```

Comment:
+1 to operator labeling. From my perspective as "not a rail enthusiast," the operator is more interesting than the named section of track.

Reply:
Since name and operator info is not surfaced by the vector tiles, I think it'd be out of scope for this PR, but would be welcome as a future enhancement.